frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -126,7 +138,7 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
                 <PropertyItem label="Repository" value={repository || notAvailable} />
                 <PropertyItem label="Container Image" value={containerImage || notAvailable} />
                 <PropertyItem label="Created At" value={createdAt || notAvailable} />
-                <PropertyItem label="Support" value={support || notAvailable} />
+                <PropertyItem label="Support" value={supportWorkflow || notAvailable} />
```

Comment:
Will this be rendered as a link?

Reply:
Looks like it's currently just text with no linking happening. I can turn it into a link, but what would the link text be?